dali/test/python/test_eager_coverage.py

```diff
@@ -0,0 +1,985 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
I'll add documentation. Just some explanation here:
> - `data` -> what is it for? having `data`, `audio_data` and `flat_data` is somewhat ambiguous, since the .

It is a little ambiguous, but `data` is of sample "image" shape, and for the rest I guess the names more or less reflect the purpose. In particular they differ in shapes.

>- `GetData` -> here it is hard to generally grasp the purpose from the code only, why couldn't these be just a free functions?

They could be free functions, I just preferred them organized in a class.